### ISSUE TITLE
Enhance the UI of the game

### ISSUE THREAD

**Issue by grvmourya** [TusharAMD/Runn#17]
I'm currently working as a unity game developer and have worked on some cool looking games, if possible do let me handle the UI section of the game so that it be more intuitive to the end users.

**Comment by TusharAMD** [comment]
Sure. Alternatively you can also design a new game of your choice using unity.

**Comment by grvmourya** [comment]
First of all thanks tushar for assigning me. Secondly I wanted to ask is it possible that I'll develop a game on unity(based on C#) and you can implement the gesture controls on it?

**Comment by TusharAMD** [comment]
Yes that's totally fine, just make a new folder and add all the game contents into it.

**Comment by grvmourya** [comment]
Sure I'll start working on it then👍🏻
